docs/editor_manual/copying_aliasing_existing_pages.md

```diff
@@ -4,30 +4,31 @@
 
 Here is how to copy an existing page:
 
--   When you hover over a page in an {doc}`Explorer page <finding_your_way_around/the_explorer_page>`, click on _More_ and then _Copy_.
+-   Hover over a page in an {doc}`Explorer page <finding_your_way_around/the_explorer_page>`, click on _More_ and then _Copy_.
 
 ![Copy action available when hovering over a page in an explorer page](../_static/images/screen12.6_1_copy_from_explorer_menu.png)
 
 -   You are then taken to a form where you can enter the copy's title, slug and choose the parent page (if you want to change it). You then get the option to publish the copied page right away and an option to mark the copy as an alias of the original page (see [](aliasing_page_label) for more information about aliasing pages).
 
 ![Copy page form with the options to change the title, slug, parent page, published status and option to create an alias.](../_static/images/screen12.6_2_copy_input_information.png)
 
--   Once you completed this form, press _Copy this page_. The page will then be copied and appear in the Explorer page.
+-   Once you have completed this form, press _Copy this page_. The page will then be copied and appear in the Explorer page.
 
 (aliasing_page_label)=
 
 ## Aliasing pages
 
 When copying a page, you also have the option to mark it as an alias. The content of an aliased page will stay in sync with the original. This is particularly useful when a page is required to be available in multiple places.
-For example, let's say you have a page about Brioche in the Breads section. You then want to also make this page available in the Pastries section. One way to do this is to create a copy of the Brioche page and change the parent page to the Pastries page. However you now need to remember to update this copy each time the original is modified. If a copy is marked as an alias Wagtail will do this for you each time a modification to the original page is published.
+For example, let's say you have a page about _Brioche_ in the **Breads section**. You then want to also make this page available in the **Pastries section**. One way to do this is to create a copy of the _Brioche_ page and change the parent page to the Pastries page. However you now need to remember to update this copy each time the original is modified. If a copy is marked as an alias, Wagtail will do this for you each time a modification to the original page is published.
 
 Here is how to create an alias to an existing page:
 
--   The first step is the same as it is for [](copying_page_label). When you hover over a page in the {doc}`Explorer page <finding_your_way_around/the_explorer_page>`, click on _More_ and then _Copy_.
+    The first step is the same as it is for [](copying_page_label). 
```

Comment:
This looks like it's missing the dash.

Reply:
Hello LB. Thank you for the feedback. I have added the dash, I only removed it because the actual steps to take didn't start on that line but the next. People scanning through the docs would rather fix their eyes on the bulleted points. I don't know if you understand my thought process.